### PR TITLE
Textbox for context

### PR DESCRIPTION
**Pull request by corinnabolon**
This input will need to be within a form, but can it also be its own separate component?  I was planning to add the button too, but that will have to be the submit button for the form, which will also include the color, so I am not sure if separating the input into its own component is really the best way to go about this... but it's created anyway!

### CODE REVIEW

File: .vscode/settings.json
Comment (lnovitz) [inline .vscode/settings.json:1]: oops, we need to add `.vscode/` to `.gitignore` file

File: colorpicker/yarn.lock
Comment (lnovitz) [inline colorpicker/yarn.lock:1]: I believe we're sticking to `npm` not `yarn` so if you can delete this file that'd be great. We only want to commit `package.json` if we change packages

File: colorpicker/src/App.jsx [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
-      )
-    }
\ No newline at end of file
+  return (
+    <>
+      <h1 className="text-3xl font-bold underline">Hello world!</h1>
+      <ContextBox />
+    </>
+  );
+}
```
Comment (lnovitz) [inline colorpicker/src/App.jsx:11]: You'll need to merge the latest changes from `Development` into your branch. 

`git fetch`
`git checkout corinna-textbox`
`git merge Development`

You'll have to resolve merge conflicts. We can do this together tomorrow if you like.

**Review by lnovitz — CHANGES_REQUESTED** [review]

File: colorpicker/src/App.jsx (hunk 1 above)
Comment (corinnabolon) [reply colorpicker/src/App.jsx:11]: Yeah, I can't figure out how to resolve the conflicts!  I tried but it's still saying there are conflicts.  Hope I didn't mess it up more... I'd love help with that tomorrow.

**Pull request closed without merging**